STEERING-COMMITTEE.md

```diff
@@ -48,6 +48,37 @@ the way we run this committee, based on feedback from the community.
 1. Act as the final escalation point and decider for any disputes, issues,
    clarifications, or escalations within the project scope.
 
+## Knative Trademark Committee Charter
+
+1. Manage licensing the Knative brand and decide which things can be called
+   "Knative" and how that mark can be used in relation to other efforts or
+   vendors.
+1. Review and approve conformance rules and tests that define brand use
+   decisions.
+1. Review and approve definition of Knative Core scope.
+1. Only the Trademark Committee can redefine its own charter and authority over
+   Knative Trademark, conformance, and brand decisions.
+
+## Trademark Committee Composition
+
+The Trademark Committee is composed of company representatives from companies
+which have substantially contributed to Knative over the last year. For the 2020
+year, one seat is assigned to each of Google, IBM/RedHat, and VMware. The
+committee shall review this composition at least annually and update these
```

Comment:
Is this Trademark Committee here? When it says at least annually, it might be good to discuss what triggers the review, or what the process of initiating. Also deets on how this composition is determined.

Reply:
I think this is fine for the first year.

Reply:
The intro sentence should be clarified to: "The Trademark Committee is composed of company representatives from companies who have substantially contributed to Knative, as well as the Trademark holder."

Reply:
Should we consider a more distinct name for the Trademark Committee, so that we're never confused about which body we're speaking of? Perhaps "Trademark Board", or some other name, will make it so that we're always clear if we're referring to KTC or to SC or to TOC.

Reply:
> The intro sentence should be clarified to: "The Trademark Committee is composed of company representatives from companies who have substantially contributed to Knative, as well as the Trademark holder."

I understand why it is important for the mark holder (as a vendor) to have presence on this body, but it seems like a dysfunctional world where Google holds the mark while the rest of the community makes all the substantial contributions.  This body is intended to represent (and protect) those with substantial ongoing investment in the upstream project.  The incentives don't seem properly aligned here.

Reply:
you have to trust that if Google were to stop contributing "significantly" to Knative that they would relinquish control of the mark and would be open to change some of the wording in this doc.

Reply:
Part of my calling out this change is that it seems to directly contradict the assumption in your statement.

I'd rather not assume anything, I'd like things written down.